### PR TITLE
newdevice: technicolor TG650S and TG670S 

### PR DESCRIPTION
**Pull request by tomarch**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

---
This change is [Reviewable](https://reviewable.io/reviews/librenms/librenms/7420)

### CODE REVIEW

File: tests/snmpsim/technicolor_tg650s.snmprec [hunk 1]
```diff
@@ -0,0 +1,3 @@
+1.3.6.1.2.1.1.1.0|4|TG650s
```
Comment (laf) [inline tests/snmpsim/technicolor_tg650s.snmprec]: Is this the full sysDescr and sysObjectID from the devices?

File: includes/polling/os/technicolor.inc.php [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?php
+
+use LibreNMS\RRD\RrdDefinition;
```
Comment (laf) [inline includes/polling/os/technicolor.inc.php]: You don't need this line.

Please also add a copyright info https://docs.librenms.org/#General/Contributing/#copyright

File: includes/definitions/technicolor.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+os: technicolor
+text: 'technicolor'
```
Comment (laf) [inline includes/definitions/technicolor.yaml]: text: Should describe the device in a little more detail. Like `Cisco IOS`

File: includes/definitions/technicolor.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+os: technicolor
```
Comment (laf) [inline includes/definitions/technicolor.yaml]: The os name should be called after the firmware the device runs like IOS for Cisco IOS. Using the vendor name isn't really the best option, what can be used instead?

File: includes/polling/os/technicolor.inc.php (hunk 2 above)
Comment (tomarch) [reply includes/polling/os/technicolor.inc.php]: This was add by the new-os.php script.
Fixed

File: tests/snmpsim/technicolor_tg650s.snmprec (hunk 1 above)
Comment (tomarch) [reply tests/snmpsim/technicolor_tg650s.snmprec]: sysDescr is OK
sysObjectID is 1.3.6.1.4.1.2863.405.1

Fixed by new commit

File: includes/definitions/technicolor.yaml (hunk 4 above)
Comment (tomarch) [reply includes/definitions/technicolor.yaml]: I don't find many information about it.
I've only this:
_COMPANY_NAME=Technicolor
_COMPANY_URL=http://www.technicolor.com
_PROD_NAME=Technicolor TG
_BRAND_NAME=MediaAccess


Maybe we can call it "tg-os" or "mediaaccess-os".

File: includes/polling/os/technicolor.inc.php (hunk 2 above)
Comment (laf) [reply includes/polling/os/technicolor.inc.php]: I'll fix that in the script

Comment (laf) [reply includes/polling/os/technicolor.inc.php]: In fact, it's not. We don't create that file.

File: includes/definitions/technicolor.yaml (hunk 4 above)
Comment (laf) [reply includes/definitions/technicolor.yaml]: tgos would be ok if the TG devices run the same firmware

File: html/images/os/technicolor.svg [hunk 5]
```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (laf) [inline html/images/os/technicolor.svg]: Can you compress this svg by using: https://jakearchibald.github.io/svgomg/

File: includes/definitions/tgos.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+type: network
+icon: technicolor
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+    - { graph: device_processor, text: 'CPU Usage' }
+    - { graph: device_mempool, text: 'Memory Usage' }
+discovery:
+    - sysObjectId:
+        - .1.3.6.1.4.1.2863.405.1
+    - sysDescr_regex:
```
Comment (laf) [inline includes/definitions/tgos.yaml]: You can remove this check. Using the sysObjectId is better and preferred.

Comment (tomarch) [reply includes/definitions/tgos.yaml]: You speak about the "sysDescr_regex" part, right ?

Comment (laf) [reply includes/definitions/tgos.yaml]: Correct (and obviously the line below it).

**Comment by laf** [comment]
Thanks again :D

**Pull request merged**